flink-runtime/src/test/java/org/apache/flink/runtime/taskexecutor/TaskExecutorTest.java

```diff
@@ -1744,6 +1757,39 @@ public void testSyncSlotsWithJobMasterByHeartbeat() throws Exception {
 		}
 	}
 
+	@Test
+	public void testSlotReportResourceProfile() throws Throwable {
+		Configuration config = new Configuration();
+		config.setInteger(TaskManagerOptions.NUM_TASK_SLOTS, 2);
+		config.setString(NettyShuffleEnvironmentOptions.NETWORK_BUFFERS_MEMORY_MIN, "100m");
```

Comment:
I think it may be better to define the test data in variables, and get the target value in the following `assertEquals` with the relationship between the variables, like
<pre>
    int networkMemoryInMB = 100;
    ....
    assertEquals(networkMemoryInMB / numSlots, resourceProfile.getNetworkMemoryInMB());
</pre>